move/axelar_gateway/sources/gateway.move

```diff
@@ -31,14 +31,16 @@ use axelar_gateway::auth::{Self, AxelarSigners, validate_proof};
 use axelar_gateway::bytes32::{Self, Bytes32};
 use axelar_gateway::channel::{Self, Channel, ApprovedMessage};
 use axelar_gateway::message::{Self, Message};
-use axelar_gateway::proof::{Self, Proof};
-use axelar_gateway::weighted_signers::{Self, WeightedSigners};
+use axelar_gateway::proof;
+use axelar_gateway::weighted_signers;
 use std::ascii::String;
 use sui::address;
-use sui::bcs;
 use sui::clock::Clock;
 use sui::hash;
 use sui::table::{Self, Table};
+use utils::utils;
+#[test_only]
+use sui::bcs;
```

Comment:
can you import this just before test only section, or does the formatter move it here?